src/routes/magicSquare.ts

```diff
@@ -0,0 +1,49 @@
+// TODO @anukulpandey: remove after campaign is over
+import { Router } from 'express';
+import { sequelize } from '../db/sequelize.db';
+import { MagicSquare } from '../db/MagicSquare.db';
+MagicSquare
+
+const router = Router();
+
+interface MagicSquareParams {
+    vid:string;
+    eventId:string;
+    address:string;
+    eventsCount:number;
+}
+
+router.post('/', async (req, res) => {
+    try {
+        const { vid, eventId, address, eventsCount }:MagicSquareParams = req.body;
+        let magicSquareRecord = await sequelize.getRepository(MagicSquare).findOne({
+            where: {
+                vid: vid,
+                eventId: eventId
+            }
+        });
+        if (magicSquareRecord) {
+            magicSquareRecord.eventsCount = eventsCount;
+            await magicSquareRecord.save();
+        } else {
+            await sequelize.getRepository(MagicSquare).create({
+                vid: vid,
+                eventId: eventId,
+                address: address,
+                eventsCount: eventsCount
+            } as any);
+        }
+
```

Comment:
need to make request to magicsquare api with their identifier - base url can be hardcoded